src/core/components/pin.js

```diff
@@ -66,11 +66,10 @@ module.exports = function pin (self) {
       update: (multihash, cb) => {
         try {
           multihashes.validate(multihash)
-          normalized.hashes.push(multihash)
-          cb()
-        } catch (ex) {
-          cb(ex)
-        }
+        } catch (ex) { return cb(ex) }
```

Comment:
`s/ex/err` so that any standard linter catches if this exception is not handled.

Reply:
Sorry, can you explain what you mean by `s/ex/err`?

Reply:
Ah, sorry, was using [`sed`](http://www.grymoire.com/Unix/Sed.html) notation.

I meant to replace the word `ex` by the word `err`, so that the [Standard JavaScript Linter](https://standardjs.com/) recognizes it as an error that needs to be taken care of.

Reply:
Ah, ok, no problem.